tests/test_factory_constructors.cpp

```diff
@@ -88,8 +88,8 @@ class PyTF6 : public TestFactory6 {
     PyTF6(PyTF6 &&f) : TestFactory6(std::move(f)) { print_move_created(this); }
     PyTF6(const PyTF6 &f) : TestFactory6(f) { print_copy_created(this); }
     PyTF6(std::string s) : TestFactory6((int) s.size()) { alias = true; print_created(this, s); }
-    virtual ~PyTF6() { print_destroyed(this); }
-    int get() override { PYBIND11_OVERRIDE(int, TestFactory6, get, /*no args*/); }
+    ~PyTF6() override { print_destroyed(this); }
+    int get() override { PYBIND11_OVERLOAD(int, TestFactory6, get, /*no args*/); }
```

Comment:
Oops, incorrect rebase, I got it backwards. Sorry, @YannickJadoul !

Reply:
Wait, what do you mean?
What went wrong?

Reply:
Oh, now I see. It's clearer in the full diff.

Do you want to just push a commit to master? Or do you want me to make a PR?

(Basically, `s/PYBIND11_OVERLOAD/PYBIND11_OVERRIDE/` should work)

Reply:
But not on the definition :). I think I fixed it. It was only a problem where I manually rebased, because I got it backwards.

Reply:
:D Good point. But fixed by #2490!